tests/blazorserver-calls-api/Client/blazorserver-client.csproj

```diff
@@ -6,6 +6,10 @@
     <WebProject_DirectoryAccessLevelKey>0</WebProject_DirectoryAccessLevelKey>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
```

Comment:
@jmprieur i had to add this package...not sure why it was removed before or what happened...fyi.